### DISCUSSION TITLE
Atorch DL24EW wifi wersion how to add to local tuya

### DISCUSSION THREAD

**Discussion by KreeQ76** [syssi/esphome-atorch-dl24#88] [Q&A]
Hi i just have some problem to add a DL24EW to local tuya integration.

how to add a device with more than one sensor or funkcion, i am just started with HA..

<img width="162" alt="Zrzut ekranu 2024-11-25 o 20 15 30" src="https://github.com/user-attachments/assets/dc57b9c7-550d-499c-93f9-00afdc7c9343">

**Comment by syssi** [comment]
Unfortunately this project is about another generation of the dummy load. Your device (WiFi) isn't supported. The previous generation did you bluetooth low energy instead.

**Reply by KreeQ76** [reply]
ok thanks for help.
